### PR TITLE
Initial project setup, updates readme, and adds function to get api endpoints

### PR DESCRIPTION
**Pull request by PiX3L-NoMAD**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+* #### .env.test
 
+* #### .env.development
+
+#### Inside .env.test add the line:
+    PGDATABASE=nc_news_test
+
+#### Inside .env.development add the line:
+    PGDATABASE=nc_news
```
Comment (nc-gary-evans) [inline README.md:13]: What is the purpose of a .env file and why should we add them to gitignore?

File: db/data/test-data/index.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
-exports.articleData = require('./articles.js');
-exports.commentData = require('./comments.js');
-exports.topicData = require('./topics.js');
-exports.userData = require('./users.js');
+articleData = require('./articles.js');
+commentData = require('./comments.js');
+topicData = require('./topics.js');
+userData = require('./users.js');
+
+modules.export = { topicData, articleData, userData, commentData };
```
Comment (nc-gary-evans) [inline db/data/test-data/index.js:6]: What is your reasoning for changing the exports and is this working as expected?

Comment (PiX3L-NoMAD) [reply db/data/test-data/index.js:6]: I just assumed that's what I had to do when task 1 said: 

"Make sure the index file exports an object with values of the data from that folder with the keys:
topicData
articleData
userData
commentData"

I had a misplaced s, but fixed it at some point after this pull request was sent.

File: README.md (hunk 1 above)
Comment (PiX3L-NoMAD) [reply README.md:13]: Whoops! The purpose is to keep these private, the content of those files should be provided directly to the other developer in another way, or the other developer can create their own database name. I will adjust the readme file.

**Pull request merged**